package.json

```diff
@@ -35,7 +35,7 @@
     "build:compile": "babel --source-maps -d lib src",
     "build:bundle": "NODE_ENV=production webpack -p --progress",
     "build:bundle:dev": "webpack --optimize-occurence-order --progress",
-    "build:electron": "npm run clean && npm run build && cpx electron/config.json webapp/ && build -lwm",
+    "build:electron": "rimraf electron/dist && npm run clean && npm run build && build -wm --ia32 --x64 && scripts/check-electron.sh",
```

Comment:
> `build -wm --ia32 --x64`

no linux build?

Does this mean that we get 32-bit and 64-bit mac builds, or does --ia32 not apply to the mac build?